File: frontend/templates/dashboard/fix_colors.js
const fs = require('fs');
const file = 'c:/Users/Rothe/Rotherick/Laboratorio/frontend/templates/dashboard/admin_dashboard.html';
let content = fs.readFileSync(file, 'utf8');
const lines = content.split('\n');

// Replace everything between line 3633 (analyzePatientGeneral start) and the end of the script
for (let i = 3633; i < lines.length; i++) {
    if (lines[i]) {
        lines[i] = lines[i].replace(/ia-modal-content/g, 'ia-analysis-result');
        lines[i] = lines[i].replace(/color:\s*#fff/g, 'color: #333');
        lines[i] = lines[i].replace(/rgba\(255,255,255,/g, 'rgba(0,0,0,');
    }
}

fs.writeFileSync(file, lines.join('\n'));
console.log('Fixed IDs and colors successfully.');

File: frontend/templates/dashboard/fix_colors.py
import re

file_path = 'c:/Users/Rothe/Rotherick/Laboratorio/frontend/templates/dashboard/admin_dashboard.html'
with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i in range(3633, len(lines)):
    if 'ia-modal-content' in lines[i]:
        lines[i] = lines[i].replace('ia-modal-content', 'ia-analysis-result')
    if 'color: #fff' in lines[i]:
        lines[i] = re.sub(r'color:\s*#fff', 'color: #333', lines[i])
    if 'rgba(255,255,255,' in lines[i]:
        lines[i] = lines[i].replace('rgba(255,255,255,', 'rgba(0,0,0,')

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(lines)

print('Fixed IDs and colors successfully.')
